index.rst

```diff
@@ -1,58 +1,163 @@
 ..
   Technote content.
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+:tocdepth: 1
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Use the following syntax for sections:
+.. sectnum::
 
-  Sections
-  ========
+.. TODO: Delete the note below before merging new content to the master branch.
 
-  and
+.. note::
 
-  Subsections
-  -----------
+   **This is a SQuaRE delivery note covering use of the Science Platform in Observatory Operations (not the Data Facilities).**
 
-  and
+   A Delivery Note is being used to transition a service (or other codebase) from Rubin Construction to Operations in situations where the service is not covered by a requirements document, is not part of the observatory operational readiness milestone, or otherwise lacks a formal process of delivery.
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+   Its purpose is to document the transition so as to allow Ops-funded developers to continue to maintain and evolve the service, and to allow the service's deployment costs to be covered by Ops infrastructure or funding. It can also be used to describe responsibilities for the service when not a direct continuation of the model used in construction.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+   The process is complete when the relevant AD(s) and/or Director reviews the note and the note is published (merged to master).
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+   * **Engineering review:** Michael Reuter
 
-     Caption text.
+   * **Management review:** Wil O'Mullane, Chuck Claver and Bob Blum
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+.. Add content here.
 
-   Feel free to delete this instructional comment.
+Scope
+=====
 
-:tocdepth: 1
+The scope of this Delivery Note is usage of the Rubin Science Platform (including) associated infrastructure for observatory operations, including (but not limited to) control of the observing queue, analysis and visualisation of incoming telescope data and analsysis of telemetry and other quantities stored in the Engineering Facilities Database.
```

Comment:
I think the parentheses around _including_ should be removed as it feels important to the sentence, but that's just MHO.